nutrition_labels/grant_tagger_evaluation.py

```diff
@@ -108,12 +108,12 @@ def get_model_scores_df(config, args):
     model_date_dir = os.path.join('models', config_version)
 
     model_names = os.listdir(f'{model_date_dir}')
-    model_names.remove('training_information.json')
-    model_names.remove('.DS_Store')
-    try:
-        model_names.remove('test_data_formatted.csv')
-    except:
-        print('test_data_formatted.csv not created yet')
+
+    # Remove items in this directory which aren't model names
+    not_model_names = ['training_information.json', '.DS_Store', 'test_data_formatted.csv']
+    for not_model_name in not_model_names:
+        if not_model_name in model_names:
+            model_names.remove(not_model_name)
```

Comment:
list comprehension would be more concise? `model_names = [model for model in model_name is model not in not_model_names]`?

Also, I feel like a "positive" logic would be more readable, i.e. what are the files that are actually models? (`.ini,?`)

Reply:
yep very good point!